[2023-03-11T08:46:29.012+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: snf_dag2.create_lineitem_table manual__2023-03-11T08:46:28.551396+00:00 [queued]>
[2023-03-11T08:46:29.019+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: snf_dag2.create_lineitem_table manual__2023-03-11T08:46:28.551396+00:00 [queued]>
[2023-03-11T08:46:29.020+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-11T08:46:29.021+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2023-03-11T08:46:29.021+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-11T08:46:29.029+0000] {taskinstance.py:1300} INFO - Executing <Task(SnowflakeOperator): create_lineitem_table> on 2023-03-11 08:46:28.551396+00:00
[2023-03-11T08:46:29.034+0000] {standard_task_runner.py:55} INFO - Started process 3523 to run task
[2023-03-11T08:46:29.038+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'snf_dag2', 'create_lineitem_table', 'manual__2023-03-11T08:46:28.551396+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/snf_dag_2.py', '--cfg-path', '/tmp/tmp0ahr6ahr']
[2023-03-11T08:46:29.039+0000] {standard_task_runner.py:83} INFO - Job 51: Subtask create_lineitem_table
[2023-03-11T08:46:29.094+0000] {task_command.py:388} INFO - Running <TaskInstance: snf_dag2.create_lineitem_table manual__2023-03-11T08:46:28.551396+00:00 [running]> on host 1e740946fd0d
[2023-03-11T08:46:29.143+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=snf_dag2
AIRFLOW_CTX_TASK_ID=create_lineitem_table
AIRFLOW_CTX_EXECUTION_DATE=2023-03-11T08:46:28.551396+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-11T08:46:28.551396+00:00
[2023-03-11T08:46:29.144+0000] {sql.py:254} INFO - Executing: create or replace table lineitem cluster by (L_SHIPDATE) 
as select * from "SNOWFLAKE_SAMPLE_DATA"."TPCH_SF100"."LINEITEM" limit 500;
[2023-03-11T08:46:29.153+0000] {base.py:73} INFO - Using connection ID 'snowflake_default' for task execution.
[2023-03-11T08:46:29.536+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/snowflake/connector/options.py:108: UserWarning: You have an incompatible version of 'pyarrow' installed (9.0.0), please install a version that adheres to: 'pyarrow<10.1.0,>=10.0.1; extra == "pandas"'
  "pyarrow", installed_pyarrow_version, pandas_pyarrow_extra

[2023-03-11T08:46:29.844+0000] {base.py:73} INFO - Using connection ID 'snowflake_default' for task execution.
[2023-03-11T08:46:29.846+0000] {connection.py:287} INFO - Snowflake Connector for Python Version: 3.0.1, Python Version: 3.7.16, Platform: Linux-5.15.49-linuxkit-aarch64-with-debian-11.6
[2023-03-11T08:46:29.846+0000] {connection.py:990} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2023-03-11T08:46:29.847+0000] {connection.py:1007} INFO - Setting use_openssl_only mode to False
[2023-03-11T08:46:30.387+0000] {cursor.py:738} INFO - query: [ALTER SESSION SET autocommit=False]
[2023-03-11T08:46:30.472+0000] {cursor.py:751} INFO - query execution done
[2023-03-11T08:46:30.473+0000] {cursor.py:891} INFO - Number of results in first chunk: 1
[2023-03-11T08:46:30.474+0000] {sql.py:375} INFO - Running statement: create or replace table lineitem cluster by (L_SHIPDATE) 
as select * from "SNOWFLAKE_SAMPLE_DATA"."TPCH_SF100"."LINEITEM" limit 500;, parameters: None
[2023-03-11T08:46:30.475+0000] {cursor.py:738} INFO - query: [create or replace table lineitem cluster by (L_SHIPDATE) as select * from "SNOWF...]
[2023-03-11T08:46:31.715+0000] {cursor.py:751} INFO - query execution done
[2023-03-11T08:46:31.722+0000] {cursor.py:891} INFO - Number of results in first chunk: 1
[2023-03-11T08:46:31.727+0000] {sql.py:384} INFO - Rows affected: 1
[2023-03-11T08:46:31.730+0000] {snowflake.py:391} INFO - Rows affected: 1
[2023-03-11T08:46:31.732+0000] {snowflake.py:392} INFO - Snowflake query id: 01aadeee-0000-4fb4-0004-9dfe00014112
[2023-03-11T08:46:31.733+0000] {cursor.py:738} INFO - query: [COMMIT]
[2023-03-11T08:46:31.794+0000] {cursor.py:751} INFO - query execution done
[2023-03-11T08:46:31.795+0000] {cursor.py:891} INFO - Number of results in first chunk: 1
[2023-03-11T08:46:31.796+0000] {connection.py:586} INFO - closed
[2023-03-11T08:46:31.838+0000] {connection.py:589} INFO - No async queries seem to be running, deleting session
[2023-03-11T08:46:31.945+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=snf_dag2, task_id=create_lineitem_table, execution_date=20230311T084628, start_date=20230311T084629, end_date=20230311T084631
[2023-03-11T08:46:32.023+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-11T08:46:32.039+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
